src/components/dialogs/generator-creation-dialog.js

```diff
@@ -460,86 +489,76 @@ const GeneratorCreationDialog = ({
                             {gridItem(generatorNameField, 4)}
                             {gridItem(energySourceField, 4)}
                         </Grid>
+
                         {/* Connectivity part */}
-                        <Grid container spacing={2}>
-                            <Grid item xs={12}>
-                                <h3 className={classes.h3}>
-                                    <FormattedMessage id="Connectivity" />
-                                </h3>
-                            </Grid>
-                        </Grid>
+                        <GridSection title="Connectivity" />
```

Comment:
it's a translation IDthere,  then it should be clearer for the propName
maybe something like `titleMessageDescriptor` with this format :
https://formatjs.io/docs/react-intl/components/#formattedmessage
then you pass it throught FormattedMessage component

Reply:
`GridSubSection` is based on the old `GridSection` that was already there.
I could either change the old code and refactor the eight files that use GridSection, or I could use the same templating as GridSection.
What do you prefer ?